analyzers/src/SonarAnalyzer.Common/Rules/TooManyParametersBase.cs

```diff
@@ -72,9 +72,13 @@ protected static bool VerifyCanBeChangedBySymbol(SyntaxNode node, SemanticModel
                 return true;    // Not a declaration, such as Action
             }
 
-            if (declaredSymbol.IsExtern && declaredSymbol.IsStatic && declaredSymbol.HasAttribute(KnownType.System_Runtime_InteropServices_DllImportAttribute))
+            if (declaredSymbol.IsStatic)
```

Comment:
should the two `if`s be merged?

Reply:
I've seen the code smell but I didn't like the mixture of all these `&&` and `||`. I think it is better readable this way but I wouldn't mind changing it if you prefer it the other way around.

Reply:
I see your point